src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+
+  @for $index from 1 through 31 {
+    &__day--#{$index} {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      border: 1px solid black;
+      box-sizing: border-box;
+      width: $colWidth;
+      height: 100px;
+
+      background-color: #eee;
+
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      text-align: center;
```

Comment:
These files are common, no need to add them in the loop. Check the size of resulting css file, it will be enormous.